### ISSUE TITLE
Support SRC-20 art

### ISSUE THREAD

**Issue by brandonmarshall-tm** [blockstack/blockstack-app#5311]
At our request, the SRC20 API was recently updated to add a `deploy_img` endpoint, which corresponds with the SRC-20's art.

The accompanying SRC20 image is an important part of the Stamp culture, and part of what makes trading them interesting. If one is available, it would be great to show them in place of the default SRC-20 icon.

Some examples:
![image](https://github.com/leather-wallet/extension/assets/122313287/f095f2cf-c4c5-4b3f-8eb7-e96b58f084a7)
![image](https://github.com/leather-wallet/extension/assets/122313287/bf17ca75-787c-4115-b684-a8afd3b22527)

SRC-20 marketplaces usually display the art with the corresponding JSON overlayed on top like this:
![Screenshot 2024-04-24 at 10 30 24 AM](https://github.com/leather-wallet/extension/assets/122313287/b9b5a118-f3d0-4290-bfad-e73b45519678)

**Comment by markmhendrickson** [comment]
Mind commenting with a few sample asset URLs for reference?

I'm wondering how we ought to format these assets if we use them as token icons, since the artwork is usually in the bottom-right quadrant and will get cut off if we simply apply a round silhouette?

**Comment by brandonmarshall-tm** [comment]
Agreed, we'd need to format as a square since so much of the art is around the outside of the image to make room for the JSON text in the middle. Pretty much every SRC20 token has its own art: https://openstamp.io/market/src20
